  INFO root [0x000020bc] [2018-06-20 10:57:54,147] - 
  INFO root [0x000020bc] [2018-06-20 10:57:54,148] - -----------------------------------------------------------------------------------------


  INFO root [0x000020bc] [2018-06-20 10:57:54,150] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x000020bc] [2018-06-20 10:57:54,150] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x000020bc] [2018-06-20 10:57:54,151] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x000020bc] [2018-06-20 10:57:54,164] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x000020bc] [2018-06-20 10:57:54,165] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x000020bc] [2018-06-20 10:57:54,169] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:3558 )
  INFO root [0x000020bc] [2018-06-20 10:57:54,169] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:3558, SOMAXCONN=128.
  INFO root [0x000020bc] [2018-06-20 10:57:54,169] - server-configs:
  INFO root [0x000020bc] [2018-06-20 10:57:54,170] - 	gameUpdateHertz : 10
  INFO root [0x000020bc] [2018-06-20 10:57:54,170] - 	entryScriptFile : kbemain
  INFO root [0x000020bc] [2018-06-20 10:57:54,170] - 	internalAddr : 192.168.31.56:3558
  INFO root [0x000020bc] [2018-06-20 10:57:54,170] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x000020bc] [2018-06-20 10:57:54,171] - 	componentID : 7001
  INFO root [0x000020bc] [2018-06-20 10:57:54,788] - Components::findLogger: found logger, addr:192.168.31.56:3557
  INFO root [0x000020bc] [2018-06-20 10:57:54,788] - Components::addComponent: reset orderLog, uid=20076!
  INFO root [0x000020bc] [2018-06-20 10:57:54,789] - Components::addComponent[logger], uid=20076, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000020bc] [2018-06-20 10:57:54,798] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 20076. PID: 4672 ----
  INFO root [0x000020bc] [2018-06-20 10:57:54,798] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x000020bc] [2018-06-20 10:57:54,799] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000020bc] [2018-06-20 10:57:54,799] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000020bc] [2018-06-20 10:57:54,944] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000020bc] [2018-06-20 10:57:54,945] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000020bc] [2018-06-20 10:57:54,946] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000020bc] [2018-06-20 10:57:54,952] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000020bc] [2018-06-20 10:57:54,952] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000020bc] [2018-06-20 10:57:54,952] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000020bc] [2018-06-20 10:57:54,953] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000020bc] [2018-06-20 10:57:55,013] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000020bc] [2018-06-20 10:57:55,079] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x000020bc] [2018-06-20 10:57:55,079] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000020bc] [2018-06-20 10:57:55,106] - loaded script:Account(1).
  INFO root [0x000020bc] [2018-06-20 10:57:55,107] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000020bc] [2018-06-20 10:57:55,108] - TelnetServer server is running on port 40000
  INFO root [0x000020bc] [2018-06-20 10:57:56,270] - ---- baseapp is running ----
 DEBUG root [0x000020bc] [2018-06-20 10:57:56,299] - Components::process(): Request for the process of identity...
  WARN root [0x000020bc] [2018-06-20 10:58:02,536] - Forwarding logs to logger[192.168.31.56:3557/0/0/0]...
 ERROR root [0x000020bc] [2018-06-20 10:58:33,731] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000020bc] [2018-06-20 10:58:33,748] - The following logs sent to logger failed:
  WARN root [0x000020bc] [2018-06-20 10:58:33,765] - ==> [2018-06-20 10:58:33 715] TCPPacketReceiver::processPendingEvents(192.168.31.56:3557): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000020bc] [2018-06-20 10:58:33,765] - ==> [2018-06-20 10:58:33 731] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x000020bc] [2018-06-20 10:58:33,841] - TCPPacketReceiver::processPendingEvents(192.168.31.56:3555): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000020bc] [2018-06-20 10:58:33,843] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x000020bc] [2018-06-20 10:58:33,856] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x000020bc] [2018-06-20 10:58:33,864] - onBaseAppShutDown: state=0
  INFO root [0x000020bc] [2018-06-20 10:58:33,865] - Shutdowner::onShutdown: shutting down
S_INFO root [0x000020bc] [2018-06-20 10:58:33,865] - onBaseAppShutDown: state=1
  INFO root [0x000020bc] [2018-06-20 10:58:33,866] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x000020bc] [2018-06-20 10:58:33,866] - onBaseAppShutDown: state=2
 DEBUG root [0x000020bc] [2018-06-20 10:58:33,921] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000020bc] [2018-06-20 10:58:33,921] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000020bc] [2018-06-20 10:58:33,921] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000020bc] [2018-06-20 10:58:33,921] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000020bc] [2018-06-20 10:58:33,922] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000020bc] [2018-06-20 10:58:33,922] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000020bc] [2018-06-20 10:58:33,922] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000020bc] [2018-06-20 10:58:33,922] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x000020bc] [2018-06-20 10:58:33,938] - Script::uninstall(): is successfully!
 DEBUG root [0x000020bc] [2018-06-20 10:58:33,938] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00000c14] [2018-06-20 10:58:51,553] - 
  INFO root [0x00000c14] [2018-06-20 10:58:51,554] - -----------------------------------------------------------------------------------------


  INFO root [0x00000c14] [2018-06-20 10:58:51,556] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00000c14] [2018-06-20 10:58:51,556] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00000c14] [2018-06-20 10:58:51,557] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00000c14] [2018-06-20 10:58:51,586] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00000c14] [2018-06-20 10:58:51,587] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00000c14] [2018-06-20 10:58:51,591] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:3596 )
  INFO root [0x00000c14] [2018-06-20 10:58:51,592] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:3596, SOMAXCONN=128.
  INFO root [0x00000c14] [2018-06-20 10:58:51,592] - server-configs:
  INFO root [0x00000c14] [2018-06-20 10:58:51,592] - 	gameUpdateHertz : 10
  INFO root [0x00000c14] [2018-06-20 10:58:51,593] - 	entryScriptFile : kbemain
  INFO root [0x00000c14] [2018-06-20 10:58:51,593] - 	internalAddr : 192.168.31.56:3596
  INFO root [0x00000c14] [2018-06-20 10:58:51,593] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00000c14] [2018-06-20 10:58:51,593] - 	componentID : 7001
  INFO root [0x00000c14] [2018-06-20 10:58:52,731] - Components::findLogger: found logger, addr:192.168.31.56:3601
  INFO root [0x00000c14] [2018-06-20 10:58:52,732] - Components::addComponent: reset orderLog, uid=20266!
  INFO root [0x00000c14] [2018-06-20 10:58:52,732] - Components::addComponent[logger], uid=20266, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000c14] [2018-06-20 10:58:52,740] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 20266. PID: 5360 ----
  INFO root [0x00000c14] [2018-06-20 10:58:52,741] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00000c14] [2018-06-20 10:58:52,741] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00000c14] [2018-06-20 10:58:52,742] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00000c14] [2018-06-20 10:58:52,890] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00000c14] [2018-06-20 10:58:52,891] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00000c14] [2018-06-20 10:58:52,892] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00000c14] [2018-06-20 10:58:52,897] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00000c14] [2018-06-20 10:58:52,898] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00000c14] [2018-06-20 10:58:52,898] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00000c14] [2018-06-20 10:58:52,899] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00000c14] [2018-06-20 10:58:52,957] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00000c14] [2018-06-20 10:58:53,015] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00000c14] [2018-06-20 10:58:53,016] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00000c14] [2018-06-20 10:58:53,048] - loaded script:Account(1).
  INFO root [0x00000c14] [2018-06-20 10:58:53,048] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00000c14] [2018-06-20 10:58:53,050] - TelnetServer server is running on port 40000
  INFO root [0x00000c14] [2018-06-20 10:58:54,465] - ---- baseapp is running ----
 DEBUG root [0x00000c14] [2018-06-20 10:58:54,466] - Components::process(): Request for the process of identity...
  WARN root [0x00000c14] [2018-06-20 10:59:00,713] - Forwarding logs to logger[192.168.31.56:3601/0/0/0]...
 ERROR root [0x00000c14] [2018-06-20 14:22:26,148] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00000c14] [2018-06-20 14:22:26,160] - The following logs sent to logger failed:
  WARN root [0x00000c14] [2018-06-20 14:22:26,160] - ==> [2018-06-20 14:22:26 144] TCPPacketReceiver::processPendingEvents(192.168.31.56:3601): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000c14] [2018-06-20 14:22:26,161] - ==> [2018-06-20 14:22:26 149] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00000c14] [2018-06-20 14:22:26,262] - TCPPacketReceiver::processPendingEvents(192.168.31.56:3595): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000c14] [2018-06-20 14:22:26,269] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00000c14] [2018-06-20 14:22:26,302] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00000c14] [2018-06-20 14:22:26,303] - onBaseAppShutDown: state=0
  INFO root [0x00000c14] [2018-06-20 14:22:26,304] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00000c14] [2018-06-20 14:22:26,304] - onBaseAppShutDown: state=1
  INFO root [0x00000c14] [2018-06-20 14:22:26,304] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00000c14] [2018-06-20 14:22:26,305] - onBaseAppShutDown: state=2
 DEBUG root [0x00000c14] [2018-06-20 14:22:26,334] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00000c14] [2018-06-20 14:22:26,335] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00000c14] [2018-06-20 14:22:26,335] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00000c14] [2018-06-20 14:22:26,335] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00000c14] [2018-06-20 14:22:26,337] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00000c14] [2018-06-20 14:22:26,337] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00000c14] [2018-06-20 14:22:26,337] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00000c14] [2018-06-20 14:22:26,338] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00000c14] [2018-06-20 14:22:26,358] - Script::uninstall(): is successfully!
 DEBUG root [0x00000c14] [2018-06-20 14:22:26,359] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00001088] [2018-06-20 14:22:41,031] - 
  INFO root [0x00001088] [2018-06-20 14:22:41,032] - -----------------------------------------------------------------------------------------


  INFO root [0x00001088] [2018-06-20 14:22:41,034] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001088] [2018-06-20 14:22:41,035] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001088] [2018-06-20 14:22:41,035] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001088] [2018-06-20 14:22:41,053] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00001088] [2018-06-20 14:22:41,054] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00001088] [2018-06-20 14:22:41,058] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:7662 )
  INFO root [0x00001088] [2018-06-20 14:22:41,058] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:7662, SOMAXCONN=128.
  INFO root [0x00001088] [2018-06-20 14:22:41,059] - server-configs:
  INFO root [0x00001088] [2018-06-20 14:22:41,059] - 	gameUpdateHertz : 10
  INFO root [0x00001088] [2018-06-20 14:22:41,059] - 	entryScriptFile : kbemain
  INFO root [0x00001088] [2018-06-20 14:22:41,060] - 	internalAddr : 192.168.31.56:7662
  INFO root [0x00001088] [2018-06-20 14:22:41,060] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00001088] [2018-06-20 14:22:41,060] - 	componentID : 7001
  INFO root [0x00001088] [2018-06-20 14:22:41,869] - Components::findLogger: found logger, addr:192.168.31.56:7659
  INFO root [0x00001088] [2018-06-20 14:22:41,870] - Components::addComponent: reset orderLog, uid=27432!
  INFO root [0x00001088] [2018-06-20 14:22:41,870] - Components::addComponent[logger], uid=27432, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001088] [2018-06-20 14:22:41,877] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 27432. PID: 6292 ----
  INFO root [0x00001088] [2018-06-20 14:22:41,878] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001088] [2018-06-20 14:22:41,879] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001088] [2018-06-20 14:22:41,879] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001088] [2018-06-20 14:22:42,027] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001088] [2018-06-20 14:22:42,028] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001088] [2018-06-20 14:22:42,029] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001088] [2018-06-20 14:22:42,035] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001088] [2018-06-20 14:22:42,036] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001088] [2018-06-20 14:22:42,036] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001088] [2018-06-20 14:22:42,038] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001088] [2018-06-20 14:22:42,100] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001088] [2018-06-20 14:22:42,156] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001088] [2018-06-20 14:22:42,157] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001088] [2018-06-20 14:22:42,169] - loaded script:Account(1).
  INFO root [0x00001088] [2018-06-20 14:22:42,169] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001088] [2018-06-20 14:22:42,170] - TelnetServer server is running on port 40000
  INFO root [0x00001088] [2018-06-20 14:22:43,562] - ---- baseapp is running ----
 DEBUG root [0x00001088] [2018-06-20 14:22:43,564] - Components::process(): Request for the process of identity...
  WARN root [0x00001088] [2018-06-20 14:22:49,710] - Forwarding logs to logger[192.168.31.56:7659/0/0/0]...
 ERROR root [0x00001088] [2018-06-20 14:23:01,209] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001088] [2018-06-20 14:23:01,244] - The following logs sent to logger failed:
  WARN root [0x00001088] [2018-06-20 14:23:01,245] - ==> [2018-06-20 14:23:01 206] TCPPacketReceiver::processPendingEvents(192.168.31.56:7659): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001088] [2018-06-20 14:23:01,245] - ==> [2018-06-20 14:23:01 210] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001088] [2018-06-20 14:23:01,333] - TCPPacketReceiver::processPendingEvents(192.168.31.56:7661): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001088] [2018-06-20 14:23:01,339] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001088] [2018-06-20 14:23:01,357] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001088] [2018-06-20 14:23:01,362] - onBaseAppShutDown: state=0
  INFO root [0x00001088] [2018-06-20 14:23:01,363] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001088] [2018-06-20 14:23:01,363] - onBaseAppShutDown: state=1
  INFO root [0x00001088] [2018-06-20 14:23:01,363] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001088] [2018-06-20 14:23:01,364] - onBaseAppShutDown: state=2
 DEBUG root [0x00001088] [2018-06-20 14:23:01,397] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001088] [2018-06-20 14:23:01,397] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001088] [2018-06-20 14:23:01,398] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001088] [2018-06-20 14:23:01,398] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001088] [2018-06-20 14:23:01,398] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001088] [2018-06-20 14:23:01,399] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001088] [2018-06-20 14:23:01,399] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001088] [2018-06-20 14:23:01,399] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001088] [2018-06-20 14:23:01,419] - Script::uninstall(): is successfully!
 DEBUG root [0x00001088] [2018-06-20 14:23:01,420] - ThreadPool::destroy(): starting size 2.
  INFO root [0x000027e0] [2018-06-20 14:23:18,644] - 
  INFO root [0x000027e0] [2018-06-20 14:23:18,645] - -----------------------------------------------------------------------------------------


  INFO root [0x000027e0] [2018-06-20 14:23:18,647] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x000027e0] [2018-06-20 14:23:18,648] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x000027e0] [2018-06-20 14:23:18,648] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x000027e0] [2018-06-20 14:23:18,682] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x000027e0] [2018-06-20 14:23:18,682] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x000027e0] [2018-06-20 14:23:18,686] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:7697 )
  INFO root [0x000027e0] [2018-06-20 14:23:18,687] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:7697, SOMAXCONN=128.
  INFO root [0x000027e0] [2018-06-20 14:23:18,687] - server-configs:
  INFO root [0x000027e0] [2018-06-20 14:23:18,688] - 	gameUpdateHertz : 10
  INFO root [0x000027e0] [2018-06-20 14:23:18,688] - 	entryScriptFile : kbemain
  INFO root [0x000027e0] [2018-06-20 14:23:18,688] - 	internalAddr : 192.168.31.56:7697
  INFO root [0x000027e0] [2018-06-20 14:23:18,689] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x000027e0] [2018-06-20 14:23:18,689] - 	componentID : 7001
  INFO root [0x000027e0] [2018-06-20 14:23:19,641] - Components::findLogger: found logger, addr:192.168.31.56:7693
  INFO root [0x000027e0] [2018-06-20 14:23:19,641] - Components::addComponent: reset orderLog, uid=27557!
  INFO root [0x000027e0] [2018-06-20 14:23:19,642] - Components::addComponent[logger], uid=27557, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000027e0] [2018-06-20 14:23:19,649] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 27557. PID: 6972 ----
  INFO root [0x000027e0] [2018-06-20 14:23:19,650] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x000027e0] [2018-06-20 14:23:19,650] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000027e0] [2018-06-20 14:23:19,651] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000027e0] [2018-06-20 14:23:19,795] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000027e0] [2018-06-20 14:23:19,795] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000027e0] [2018-06-20 14:23:19,796] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000027e0] [2018-06-20 14:23:19,800] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000027e0] [2018-06-20 14:23:19,801] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000027e0] [2018-06-20 14:23:19,801] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000027e0] [2018-06-20 14:23:19,802] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000027e0] [2018-06-20 14:23:19,853] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000027e0] [2018-06-20 14:23:19,904] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x000027e0] [2018-06-20 14:23:19,905] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000027e0] [2018-06-20 14:23:19,916] - loaded script:Account(1).
  INFO root [0x000027e0] [2018-06-20 14:23:19,916] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000027e0] [2018-06-20 14:23:19,917] - TelnetServer server is running on port 40000
  INFO root [0x000027e0] [2018-06-20 14:23:20,790] - ---- baseapp is running ----
 DEBUG root [0x000027e0] [2018-06-20 14:23:20,790] - Components::process(): Request for the process of identity...
  WARN root [0x000027e0] [2018-06-20 14:23:27,151] - Forwarding logs to logger[192.168.31.56:7693/0/0/0]...
 ERROR root [0x000027e0] [2018-06-20 19:26:38,835] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000027e0] [2018-06-20 19:26:38,839] - The following logs sent to logger failed:
  WARN root [0x000027e0] [2018-06-20 19:26:38,840] - ==> [2018-06-20 19:26:38 827] TCPPacketReceiver::processPendingEvents(192.168.31.56:7693): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000027e0] [2018-06-20 19:26:38,841] - ==> [2018-06-20 19:26:38 835] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x000027e0] [2018-06-20 19:26:38,940] - TCPPacketReceiver::processPendingEvents(192.168.31.56:7696): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000027e0] [2018-06-20 19:26:38,944] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x000027e0] [2018-06-20 19:26:38,955] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x000027e0] [2018-06-20 19:26:38,959] - onBaseAppShutDown: state=0
  INFO root [0x000027e0] [2018-06-20 19:26:38,960] - Shutdowner::onShutdown: shutting down
S_INFO root [0x000027e0] [2018-06-20 19:26:38,960] - onBaseAppShutDown: state=1
  INFO root [0x000027e0] [2018-06-20 19:26:38,960] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x000027e0] [2018-06-20 19:26:38,961] - onBaseAppShutDown: state=2
 DEBUG root [0x000027e0] [2018-06-20 19:26:38,992] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000027e0] [2018-06-20 19:26:38,992] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000027e0] [2018-06-20 19:26:38,992] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000027e0] [2018-06-20 19:26:38,993] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000027e0] [2018-06-20 19:26:38,993] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000027e0] [2018-06-20 19:26:38,993] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000027e0] [2018-06-20 19:26:38,994] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000027e0] [2018-06-20 19:26:38,994] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x000027e0] [2018-06-20 19:26:39,011] - Script::uninstall(): is successfully!
 DEBUG root [0x000027e0] [2018-06-20 19:26:39,012] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00001730] [2018-06-20 19:26:57,347] - 
  INFO root [0x00001730] [2018-06-20 19:26:57,361] - -----------------------------------------------------------------------------------------


  INFO root [0x00001730] [2018-06-20 19:26:57,363] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001730] [2018-06-20 19:26:57,363] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001730] [2018-06-20 19:26:57,364] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001730] [2018-06-20 19:26:57,378] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00001730] [2018-06-20 19:26:57,379] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00001730] [2018-06-20 19:26:57,382] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:10544 )
  INFO root [0x00001730] [2018-06-20 19:26:57,382] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:10544, SOMAXCONN=128.
  INFO root [0x00001730] [2018-06-20 19:26:57,383] - server-configs:
  INFO root [0x00001730] [2018-06-20 19:26:57,383] - 	gameUpdateHertz : 10
  INFO root [0x00001730] [2018-06-20 19:26:57,383] - 	entryScriptFile : kbemain
  INFO root [0x00001730] [2018-06-20 19:26:57,383] - 	internalAddr : 192.168.31.56:10544
  INFO root [0x00001730] [2018-06-20 19:26:57,383] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00001730] [2018-06-20 19:26:57,383] - 	componentID : 7001
  INFO root [0x00001730] [2018-06-20 19:26:59,548] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 21516. PID: 8208 ----
  INFO root [0x00001730] [2018-06-20 19:26:59,549] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001730] [2018-06-20 19:26:59,550] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001730] [2018-06-20 19:26:59,551] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001730] [2018-06-20 19:26:59,773] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001730] [2018-06-20 19:26:59,774] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001730] [2018-06-20 19:26:59,775] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001730] [2018-06-20 19:26:59,782] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001730] [2018-06-20 19:26:59,783] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001730] [2018-06-20 19:26:59,784] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001730] [2018-06-20 19:26:59,785] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001730] [2018-06-20 19:26:59,921] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001730] [2018-06-20 19:27:00,082] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001730] [2018-06-20 19:27:00,083] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001730] [2018-06-20 19:27:00,619] - loaded script:Account(1).
  INFO root [0x00001730] [2018-06-20 19:27:00,620] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001730] [2018-06-20 19:27:00,621] - TelnetServer server is running on port 40000
  INFO root [0x00001730] [2018-06-20 19:27:01,812] - ---- baseapp is running ----
 DEBUG root [0x00001730] [2018-06-20 19:27:01,813] - Components::process(): Request for the process of identity...
  WARN root [0x00001730] [2018-06-20 19:27:07,956] - EntityApp::checkTickPeriod: tick took 6144% (6.14 seconds)!
  INFO root [0x00001730] [2018-06-20 19:27:09,012] - Components::findComponents: find logger(1)...
  INFO root [0x00001730] [2018-06-20 19:27:09,094] - Components::findComponents: found logger, addr:192.168.31.56:10549
  INFO root [0x00001730] [2018-06-20 19:27:09,094] - Components::addComponent: reset orderLog, uid=21516!
  INFO root [0x00001730] [2018-06-20 19:27:09,094] - Components::addComponent[logger], uid=21516, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001730] [2018-06-20 19:27:09,096] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00001730] [2018-06-20 19:27:09,197] - Components::findComponents: found dbmgr, addr:192.168.31.56:10547
  INFO root [0x00001730] [2018-06-20 19:27:09,198] - Components::addComponent: reset orderLog, uid=21516!
  INFO root [0x00001730] [2018-06-20 19:27:09,198] - Components::addComponent[dbmgr], uid=21516, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001730] [2018-06-20 19:27:09,198] - Components::findComponents: found dbmgr, addr:192.168.31.56:10547
  WARN root [0x00001730] [2018-06-20 19:27:10,199] - EntityApp::checkTickPeriod: tick took 1287% (1.29 seconds)!
  WARN root [0x00001730] [2018-06-20 19:27:10,200] - Forwarding logs to logger[192.168.31.56:10549/0/0/0]...
 ERROR root [0x00001730] [2018-06-20 19:29:30,225] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001730] [2018-06-20 19:29:30,243] - The following logs sent to logger failed:
  WARN root [0x00001730] [2018-06-20 19:29:30,244] - ==> [2018-06-20 19:29:30 219] TCPPacketReceiver::processPendingEvents(192.168.31.56:10549): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001730] [2018-06-20 19:29:30,245] - ==> [2018-06-20 19:29:30 226] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001730] [2018-06-20 19:29:30,344] - TCPPacketReceiver::processPendingEvents(192.168.31.56:10547): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001730] [2018-06-20 19:29:30,348] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001730] [2018-06-20 19:29:30,361] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001730] [2018-06-20 19:29:30,366] - onBaseAppShutDown: state=0
  INFO root [0x00001730] [2018-06-20 19:29:30,366] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001730] [2018-06-20 19:29:30,367] - onBaseAppShutDown: state=1
  INFO root [0x00001730] [2018-06-20 19:29:30,367] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001730] [2018-06-20 19:29:30,368] - onBaseAppShutDown: state=2
 ERROR root [0x00001730] [2018-06-20 19:29:30,401] - Account::onCellWriteToDBCompleted(1): not found dbmgr!
 ERROR root [0x00001730] [2018-06-20 19:29:30,402] - Base::onDestroy: not found dbmgr!
 DEBUG root [0x00001730] [2018-06-20 19:29:30,407] - Account::~Account(): 1
 DEBUG root [0x00001730] [2018-06-20 19:29:30,408] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001730] [2018-06-20 19:29:30,409] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001730] [2018-06-20 19:29:30,409] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001730] [2018-06-20 19:29:30,409] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001730] [2018-06-20 19:29:30,410] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001730] [2018-06-20 19:29:30,410] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001730] [2018-06-20 19:29:30,410] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001730] [2018-06-20 19:29:30,411] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001730] [2018-06-20 19:29:30,432] - Script::uninstall(): is successfully!
 DEBUG root [0x00001730] [2018-06-20 19:29:30,433] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00001564] [2018-06-20 19:29:44,952] - 
  INFO root [0x00001564] [2018-06-20 19:29:44,953] - -----------------------------------------------------------------------------------------


  INFO root [0x00001564] [2018-06-20 19:29:44,955] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001564] [2018-06-20 19:29:44,955] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001564] [2018-06-20 19:29:44,956] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001564] [2018-06-20 19:29:44,975] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00001564] [2018-06-20 19:29:44,976] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00001564] [2018-06-20 19:29:45,014] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:10620 )
  INFO root [0x00001564] [2018-06-20 19:29:45,016] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:10620, SOMAXCONN=128.
  INFO root [0x00001564] [2018-06-20 19:29:45,016] - server-configs:
  INFO root [0x00001564] [2018-06-20 19:29:45,016] - 	gameUpdateHertz : 10
  INFO root [0x00001564] [2018-06-20 19:29:45,016] - 	entryScriptFile : kbemain
  INFO root [0x00001564] [2018-06-20 19:29:45,017] - 	internalAddr : 192.168.31.56:10620
  INFO root [0x00001564] [2018-06-20 19:29:45,017] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00001564] [2018-06-20 19:29:45,017] - 	componentID : 7001
  INFO root [0x00001564] [2018-06-20 19:29:45,618] - Components::findLogger: found logger, addr:192.168.31.56:10615
  INFO root [0x00001564] [2018-06-20 19:29:45,619] - Components::addComponent: reset orderLog, uid=22062!
  INFO root [0x00001564] [2018-06-20 19:29:45,619] - Components::addComponent[logger], uid=22062, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001564] [2018-06-20 19:29:45,626] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 22062. PID: 11520 ----
  INFO root [0x00001564] [2018-06-20 19:29:45,627] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001564] [2018-06-20 19:29:45,627] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001564] [2018-06-20 19:29:45,628] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001564] [2018-06-20 19:29:45,777] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001564] [2018-06-20 19:29:45,778] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001564] [2018-06-20 19:29:45,780] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001564] [2018-06-20 19:29:45,786] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001564] [2018-06-20 19:29:45,787] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001564] [2018-06-20 19:29:45,787] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001564] [2018-06-20 19:29:45,788] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001564] [2018-06-20 19:29:45,851] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001564] [2018-06-20 19:29:45,915] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001564] [2018-06-20 19:29:45,915] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001564] [2018-06-20 19:29:45,959] - loaded script:Account(1).
  INFO root [0x00001564] [2018-06-20 19:29:45,960] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001564] [2018-06-20 19:29:45,961] - TelnetServer server is running on port 40000
  INFO root [0x00001564] [2018-06-20 19:29:46,759] - ---- baseapp is running ----
 DEBUG root [0x00001564] [2018-06-20 19:29:46,760] - Components::process(): Request for the process of identity...
  WARN root [0x00001564] [2018-06-20 19:29:53,131] - Forwarding logs to logger[192.168.31.56:10615/0/0/0]...
 ERROR root [0x00001564] [2018-06-20 19:31:24,363] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001564] [2018-06-20 19:31:24,364] - The following logs sent to logger failed:
  WARN root [0x00001564] [2018-06-20 19:31:24,365] - ==> [2018-06-20 19:31:24 360] TCPPacketReceiver::processPendingEvents(192.168.31.56:10615): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001564] [2018-06-20 19:31:24,365] - ==> [2018-06-20 19:31:24 364] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001564] [2018-06-20 19:31:24,524] - TCPPacketReceiver::processPendingEvents(192.168.31.56:10621): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001564] [2018-06-20 19:31:24,526] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001564] [2018-06-20 19:31:24,528] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001564] [2018-06-20 19:31:24,531] - onBaseAppShutDown: state=0
  INFO root [0x00001564] [2018-06-20 19:31:24,532] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001564] [2018-06-20 19:31:24,537] - onBaseAppShutDown: state=1
  INFO root [0x00001564] [2018-06-20 19:31:24,537] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001564] [2018-06-20 19:31:24,538] - onBaseAppShutDown: state=2
 DEBUG root [0x00001564] [2018-06-20 19:31:24,578] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001564] [2018-06-20 19:31:24,578] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001564] [2018-06-20 19:31:24,579] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001564] [2018-06-20 19:31:24,579] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001564] [2018-06-20 19:31:24,579] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001564] [2018-06-20 19:31:24,579] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001564] [2018-06-20 19:31:24,580] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001564] [2018-06-20 19:31:24,580] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001564] [2018-06-20 19:31:24,604] - Script::uninstall(): is successfully!
 DEBUG root [0x00001564] [2018-06-20 19:31:24,604] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00001a18] [2018-06-20 19:32:43,346] - 
  INFO root [0x00001a18] [2018-06-20 19:32:43,347] - -----------------------------------------------------------------------------------------


  INFO root [0x00001a18] [2018-06-20 19:32:43,349] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00001a18] [2018-06-20 19:32:43,350] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00001a18] [2018-06-20 19:32:43,352] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00001a18] [2018-06-20 19:32:43,367] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00001a18] [2018-06-20 19:32:43,368] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00001a18] [2018-06-20 19:32:43,372] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:10690 )
  INFO root [0x00001a18] [2018-06-20 19:32:43,372] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:10690, SOMAXCONN=128.
  INFO root [0x00001a18] [2018-06-20 19:32:43,373] - server-configs:
  INFO root [0x00001a18] [2018-06-20 19:32:43,373] - 	gameUpdateHertz : 10
  INFO root [0x00001a18] [2018-06-20 19:32:43,373] - 	entryScriptFile : kbemain
  INFO root [0x00001a18] [2018-06-20 19:32:43,374] - 	internalAddr : 192.168.31.56:10690
  INFO root [0x00001a18] [2018-06-20 19:32:43,374] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00001a18] [2018-06-20 19:32:43,374] - 	componentID : 7001
  INFO root [0x00001a18] [2018-06-20 19:32:44,691] - Components::findLogger: found logger, addr:192.168.31.56:10686
  INFO root [0x00001a18] [2018-06-20 19:32:44,692] - Components::addComponent: reset orderLog, uid=22646!
  INFO root [0x00001a18] [2018-06-20 19:32:44,693] - Components::addComponent[logger], uid=22646, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001a18] [2018-06-20 19:32:44,709] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 22646. PID: 3156 ----
  INFO root [0x00001a18] [2018-06-20 19:32:44,710] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00001a18] [2018-06-20 19:32:44,711] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001a18] [2018-06-20 19:32:44,712] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001a18] [2018-06-20 19:32:44,907] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001a18] [2018-06-20 19:32:44,909] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001a18] [2018-06-20 19:32:44,910] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001a18] [2018-06-20 19:32:44,917] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001a18] [2018-06-20 19:32:44,918] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001a18] [2018-06-20 19:32:44,919] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001a18] [2018-06-20 19:32:44,920] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001a18] [2018-06-20 19:32:45,011] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001a18] [2018-06-20 19:32:45,133] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001a18] [2018-06-20 19:32:45,134] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001a18] [2018-06-20 19:32:45,225] - loaded script:Account(1).
  INFO root [0x00001a18] [2018-06-20 19:32:45,226] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001a18] [2018-06-20 19:32:45,227] - TelnetServer server is running on port 40000
  INFO root [0x00001a18] [2018-06-20 19:32:46,828] - ---- baseapp is running ----
 DEBUG root [0x00001a18] [2018-06-20 19:32:46,829] - Components::process(): Request for the process of identity...
  WARN root [0x00001a18] [2018-06-20 19:32:53,156] - Forwarding logs to logger[192.168.31.56:10686/0/0/0]...
 ERROR root [0x00001a18] [2018-06-20 19:58:15,813] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001a18] [2018-06-20 19:58:15,821] - The following logs sent to logger failed:
  WARN root [0x00001a18] [2018-06-20 19:58:15,822] - ==> [2018-06-20 19:58:15 812] TCPPacketReceiver::processPendingEvents(192.168.31.56:10686): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001a18] [2018-06-20 19:58:15,823] - ==> [2018-06-20 19:58:15 814] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001a18] [2018-06-20 19:58:15,928] - TCPPacketReceiver::processPendingEvents(192.168.31.56:10692): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001a18] [2018-06-20 19:58:15,933] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001a18] [2018-06-20 19:58:15,934] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001a18] [2018-06-20 19:58:15,935] - onBaseAppShutDown: state=0
  INFO root [0x00001a18] [2018-06-20 19:58:15,935] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001a18] [2018-06-20 19:58:15,941] - onBaseAppShutDown: state=1
  INFO root [0x00001a18] [2018-06-20 19:58:15,941] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001a18] [2018-06-20 19:58:15,942] - onBaseAppShutDown: state=2
 DEBUG root [0x00001a18] [2018-06-20 19:58:15,976] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001a18] [2018-06-20 19:58:15,977] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001a18] [2018-06-20 19:58:15,977] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001a18] [2018-06-20 19:58:15,977] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001a18] [2018-06-20 19:58:15,978] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001a18] [2018-06-20 19:58:15,978] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001a18] [2018-06-20 19:58:15,978] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001a18] [2018-06-20 19:58:15,979] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001a18] [2018-06-20 19:58:15,999] - Script::uninstall(): is successfully!
 DEBUG root [0x00001a18] [2018-06-20 19:58:16,000] - ThreadPool::destroy(): starting size 2.
  INFO root [0x000029b4] [2018-06-20 19:58:27,601] - 
  INFO root [0x000029b4] [2018-06-20 19:58:27,602] - -----------------------------------------------------------------------------------------


  INFO root [0x000029b4] [2018-06-20 19:58:27,607] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x000029b4] [2018-06-20 19:58:27,608] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x000029b4] [2018-06-20 19:58:27,609] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x000029b4] [2018-06-20 19:58:27,620] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x000029b4] [2018-06-20 19:58:27,621] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x000029b4] [2018-06-20 19:58:27,624] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:10914 )
  INFO root [0x000029b4] [2018-06-20 19:58:27,624] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:10914, SOMAXCONN=128.
  INFO root [0x000029b4] [2018-06-20 19:58:27,625] - server-configs:
  INFO root [0x000029b4] [2018-06-20 19:58:27,625] - 	gameUpdateHertz : 10
  INFO root [0x000029b4] [2018-06-20 19:58:27,625] - 	entryScriptFile : kbemain
  INFO root [0x000029b4] [2018-06-20 19:58:27,625] - 	internalAddr : 192.168.31.56:10914
  INFO root [0x000029b4] [2018-06-20 19:58:27,625] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x000029b4] [2018-06-20 19:58:27,626] - 	componentID : 7001
  INFO root [0x000029b4] [2018-06-20 19:58:28,633] - Components::findLogger: found logger, addr:192.168.31.56:10912
  INFO root [0x000029b4] [2018-06-20 19:58:28,634] - Components::addComponent: reset orderLog, uid=27688!
  INFO root [0x000029b4] [2018-06-20 19:58:28,634] - Components::addComponent[logger], uid=27688, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000029b4] [2018-06-20 19:58:28,644] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 27688. PID: 7848 ----
  INFO root [0x000029b4] [2018-06-20 19:58:28,644] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x000029b4] [2018-06-20 19:58:28,645] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000029b4] [2018-06-20 19:58:28,645] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000029b4] [2018-06-20 19:58:28,799] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000029b4] [2018-06-20 19:58:28,799] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000029b4] [2018-06-20 19:58:28,800] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000029b4] [2018-06-20 19:58:28,807] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000029b4] [2018-06-20 19:58:28,807] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000029b4] [2018-06-20 19:58:28,808] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000029b4] [2018-06-20 19:58:28,810] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000029b4] [2018-06-20 19:58:28,879] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000029b4] [2018-06-20 19:58:28,940] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x000029b4] [2018-06-20 19:58:28,940] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000029b4] [2018-06-20 19:58:28,971] - loaded script:Account(1).
  INFO root [0x000029b4] [2018-06-20 19:58:28,971] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000029b4] [2018-06-20 19:58:28,973] - TelnetServer server is running on port 40000
  INFO root [0x000029b4] [2018-06-20 19:58:30,722] - ---- baseapp is running ----
 DEBUG root [0x000029b4] [2018-06-20 19:58:30,724] - Components::process(): Request for the process of identity...
  WARN root [0x000029b4] [2018-06-20 19:58:36,922] - Forwarding logs to logger[192.168.31.56:10912/0/0/0]...
 ERROR root [0x000029b4] [2018-06-20 20:00:29,460] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000029b4] [2018-06-20 20:00:29,471] - The following logs sent to logger failed:
  WARN root [0x000029b4] [2018-06-20 20:00:29,472] - ==> [2018-06-20 20:00:29 453] TCPPacketReceiver::processPendingEvents(192.168.31.56:10912): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000029b4] [2018-06-20 20:00:29,473] - ==> [2018-06-20 20:00:29 461] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x000029b4] [2018-06-20 20:00:29,554] - TCPPacketReceiver::processPendingEvents(192.168.31.56:10913): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000029b4] [2018-06-20 20:00:29,558] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x000029b4] [2018-06-20 20:00:29,569] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x000029b4] [2018-06-20 20:00:29,570] - onBaseAppShutDown: state=0
  INFO root [0x000029b4] [2018-06-20 20:00:29,570] - Shutdowner::onShutdown: shutting down
S_INFO root [0x000029b4] [2018-06-20 20:00:29,570] - onBaseAppShutDown: state=1
  INFO root [0x000029b4] [2018-06-20 20:00:29,571] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x000029b4] [2018-06-20 20:00:29,571] - onBaseAppShutDown: state=2
 DEBUG root [0x000029b4] [2018-06-20 20:00:29,601] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000029b4] [2018-06-20 20:00:29,601] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000029b4] [2018-06-20 20:00:29,601] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000029b4] [2018-06-20 20:00:29,601] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000029b4] [2018-06-20 20:00:29,601] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000029b4] [2018-06-20 20:00:29,602] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000029b4] [2018-06-20 20:00:29,602] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000029b4] [2018-06-20 20:00:29,602] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x000029b4] [2018-06-20 20:00:29,622] - Script::uninstall(): is successfully!
 DEBUG root [0x000029b4] [2018-06-20 20:00:29,623] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00002c20] [2018-06-20 22:27:46,876] - 
  INFO root [0x00002c20] [2018-06-20 22:27:46,877] - -----------------------------------------------------------------------------------------


  INFO root [0x00002c20] [2018-06-20 22:27:46,879] - Resmgr::initialize: KBE_ROOT=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/
  INFO root [0x00002c20] [2018-06-20 22:27:46,880] - Resmgr::initialize: KBE_RES_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/res/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/scripts/;C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/assets/res/
  INFO root [0x00002c20] [2018-06-20 22:27:46,881] - Resmgr::initialize: KBE_BIN_PATH=C:/Users/Public/Documents/workspace/kbe/kbengine-kbengine-v1.0.0/kbengine/kbe/bin/server/
  INFO root [0x00002c20] [2018-06-20 22:27:46,890] - NetworkInterface::recreateListeningSocket(EXTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:20015 )
  INFO root [0x00002c20] [2018-06-20 22:27:46,890] - NetworkInterface::recreateListeningSocket(EXTERNAL): address 192.168.31.56:20015, SOMAXCONN=128.
  INFO root [0x00002c20] [2018-06-20 22:27:46,893] - NetworkInterface::recreateListeningSocket(INTERNAL): bound to all interfaces with default route interface on 192.168.31.56 ( 192.168.31.56:11489 )
  INFO root [0x00002c20] [2018-06-20 22:27:46,893] - NetworkInterface::recreateListeningSocket(INTERNAL): address 192.168.31.56:11489, SOMAXCONN=128.
  INFO root [0x00002c20] [2018-06-20 22:27:46,893] - server-configs:
  INFO root [0x00002c20] [2018-06-20 22:27:46,894] - 	gameUpdateHertz : 10
  INFO root [0x00002c20] [2018-06-20 22:27:46,894] - 	entryScriptFile : kbemain
  INFO root [0x00002c20] [2018-06-20 22:27:46,894] - 	internalAddr : 192.168.31.56:11489
  INFO root [0x00002c20] [2018-06-20 22:27:46,894] - 	externalAddr : 192.168.31.56:20015
  INFO root [0x00002c20] [2018-06-20 22:27:46,894] - 	componentID : 7001
  INFO root [0x00002c20] [2018-06-20 22:27:47,748] - Components::findLogger: found logger, addr:192.168.31.56:11487
  INFO root [0x00002c20] [2018-06-20 22:27:47,748] - Components::addComponent: reset orderLog, uid=24176!
  INFO root [0x00002c20] [2018-06-20 22:27:47,749] - Components::addComponent[logger], uid=24176, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002c20] [2018-06-20 22:27:47,756] - ---- baseapp Version: 1.0.0. ScriptVersion: 0.1.0. Protocol: B363447A744C7D8D9BBF49B8B8360068. Config: Debug X86. Built: 23:18:04 Jun 15 2018. AppID: 7001. UID: 24176. PID: 7008 ----
  INFO root [0x00002c20] [2018-06-20 22:27:47,757] - Server :        Intel(R) Core(TM) i5-2415M CPU @ 2.30GHz with 4194303kB RAM
  INFO root [0x00002c20] [2018-06-20 22:27:47,757] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002c20] [2018-06-20 22:27:47,758] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002c20] [2018-06-20 22:27:47,907] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002c20] [2018-06-20 22:27:47,908] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002c20] [2018-06-20 22:27:47,909] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002c20] [2018-06-20 22:27:47,916] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002c20] [2018-06-20 22:27:47,917] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002c20] [2018-06-20 22:27:47,917] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002c20] [2018-06-20 22:27:47,918] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002c20] [2018-06-20 22:27:47,981] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002c20] [2018-06-20 22:27:48,045] - Script::install(): is successfully, Python=(3.4.2 (default, Jun 15 2018, 23:17:11) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002c20] [2018-06-20 22:27:48,045] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002c20] [2018-06-20 22:27:48,070] - loaded script:Account(1).
  INFO root [0x00002c20] [2018-06-20 22:27:48,070] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00002c20] [2018-06-20 22:27:48,072] - TelnetServer server is running on port 40000
  INFO root [0x00002c20] [2018-06-20 22:27:50,118] - ---- baseapp is running ----
 DEBUG root [0x00002c20] [2018-06-20 22:27:50,120] - Components::process(): Request for the process of identity...
  WARN root [0x00002c20] [2018-06-20 22:27:56,235] - Forwarding logs to logger[192.168.31.56:11487/0/0/0]...
 ERROR root [0x00002c20] [2018-06-20 23:41:50,939] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002c20] [2018-06-20 23:41:50,941] - The following logs sent to logger failed:
  WARN root [0x00002c20] [2018-06-20 23:41:50,942] - ==> [2018-06-20 23:41:50 936] TCPPacketReceiver::processPendingEvents(192.168.31.56:11487): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002c20] [2018-06-20 23:41:50,942] - ==> [2018-06-20 23:41:50 940] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00002c20] [2018-06-20 23:41:51,039] - TCPPacketReceiver::processPendingEvents(192.168.31.56:11483): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002c20] [2018-06-20 23:41:51,042] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00002c20] [2018-06-20 23:41:51,053] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00002c20] [2018-06-20 23:41:51,067] - onBaseAppShutDown: state=0
  INFO root [0x00002c20] [2018-06-20 23:41:51,067] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00002c20] [2018-06-20 23:41:51,068] - onBaseAppShutDown: state=1
  INFO root [0x00002c20] [2018-06-20 23:41:51,068] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00002c20] [2018-06-20 23:41:51,068] - onBaseAppShutDown: state=2
 DEBUG root [0x00002c20] [2018-06-20 23:41:51,091] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002c20] [2018-06-20 23:41:51,091] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00002c20] [2018-06-20 23:41:51,091] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00002c20] [2018-06-20 23:41:51,091] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002c20] [2018-06-20 23:41:51,091] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00002c20] [2018-06-20 23:41:51,092] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002c20] [2018-06-20 23:41:51,092] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002c20] [2018-06-20 23:41:51,092] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00002c20] [2018-06-20 23:41:51,108] - Script::uninstall(): is successfully!
 DEBUG root [0x00002c20] [2018-06-20 23:41:51,108] - ThreadPool::destroy(): starting size 2.
